### ISSUE TITLE
Translate emails on the fly

### ISSUE THREAD

**Issue by supportreq** [RainLoop/rainloop-webmail#1523]
Is there a way by which we can have translate feature like in GMail, if an email is sent in a different language it will auto translate or ask the user if they want to translate in the default language?

**Comment by djdill** [comment]
@supportreq I don't think this will ever be built in to rainloop or many other email clients apart from GMail / Hotmail. They use their own translate APIs to translate the email on the fly from the mailbox, rainloop can't access that API without paid for access.

If you want something that will work for you, I would recommend chrome translate plugin:
https://chrome.google.com/webstore/detail/google-translate/aapbdbdomjkkjkaonfhkkikfgjllcleb?hl=en

It will enable on the fly translation for you.
